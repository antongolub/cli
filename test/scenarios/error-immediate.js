'use strict'
require('../..')(main)

async function main () {
  console.log('started')
  throw new Error('boom')
  console.log('finished')
}
