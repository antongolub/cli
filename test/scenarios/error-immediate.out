started
Error: boom
    at main (scenarios/error-immediate.js:)
